gradle/package-lock.json

```diff
@@ -122,9 +122,9 @@
       },
       "dependencies": {
         "glob": {
-          "version": "7.1.4",
-          "resolved": "https://registry.npmjs.org/glob/-/glob-7.1.4.tgz",
-          "integrity": "sha512-hkLPepehmnKk41pUGm3sYxoFs/umurYfYJCerbXEyFIWcAzvpipAgVkBqqT9RBKMGjnq6kMuyYwha6csxbiM1A==",
+          "version": "7.1.6",
```

Comment:
please revert this change

Reply:
done but are You sure about that ? this is generated automatically after 'gr build'

Reply:
does not happen to me, I guess this change was caused by your (or mine) local npm/node setup